app/src/main/java/com/todoist_android/ui/auth/AuthenticationViewModel.kt

```diff
@@ -20,20 +22,13 @@ class AuthenticationViewModel(
      * and then we will use the data to update the UI
      ***/
 
-    private val _loginResponse : MutableLiveData<APIResource<LoginResponse>> = MutableLiveData()
-    private val _signUpResponse : MutableLiveData<APIResource<SignupResponse>> = MutableLiveData()
+    // use kotlin flows instead of live data
+    private val _loginResponse = MutableSharedFlow<APIResource<LoginResponse>>()
 
```

Comment:
@Njumbi what of signupResponse?

Reply:
I haven't implemented sign up ,  I don't think I have the recent changes.